app/models/cda/court_application.rb

```diff
@@ -23,6 +23,10 @@ def result_string
       Cda::CourtApplicationResultStringService.call(self)
     end
 
+    def location
+      hearing_summary.map(&:location).uniq.to_sentence
```

Comment:
silly question time: 
On a quick search for `location`, it's not clear where`location` does come from 🤔 
.. I guess it's the new method on the `application_hearing`.
Perhaps a unit test could clarify things.